chart/kubeapps/values.yaml

```diff
@@ -58,6 +62,12 @@ frontend:
     registry: docker.io
     repository: bitnami/nginx
     tag: 1.14.0-r27
+    ## Optionally specify an array of imagePullSecrets.
```

Comment:
the same here, we don't need all these repeated comments if we support only the global setting for the moment